cellprofiler/modules/plugins/measurementtemplate.py

```diff
@@ -454,8 +556,7 @@ def get_measurement_name(self, n, m):
     # of measurement columns produced. The second is more informal and
     # tells CellProfiler how to categorize its measurements.
     #
-    #
-    # get_measurement_columns gets the measurements for use in the database
+    # "get_measurement_columns" gets the measurements for use in the database
     # or in a spreadsheet. Some modules need this because they
     # might make measurements of measurements and need those names.
     #
```

Comment:
I can't add the comment on the exact line, but line 570 references `cpmeas` which should be expanded to the full name.

Reply:
There's some `np.`s in there, too.